products/public-websites/README.md

```diff
@@ -572,7 +573,7 @@ VA.gov uses search.gov as its engine. Public Websites owns that integration, and
 
 **Example content:**
 * URL: [https://www.va.gov/search/?query=test&t=false](https://www.va.gov/search/?query=test&t=false)  
-* Example issue: 
+* Example issue: TODO
```

Comment:
What should this issue be?

Reply:
Still not sure on this. I think it's ok to leave the TODO, or we can remove the whole line.

Reply:
👍